src/ragger/backend/interface.py

```diff
@@ -287,6 +293,24 @@ def both_click(self) -> None:
         """
         raise NotImplementedError
 
+    def navigate(self, instruction: NavigationInstruction) -> None:
+        """
+        Performs a click on buttons of the device depending of the navigation instruction.
+
+        :raises NotImplementedError: If the navigation instruction is not implemented.
+
+        :return: None
+        :rtype: NoneType
+        """
+        if instruction == NavigationInstruction.GO_TO_NEXT_SCREEN:
```

Comment:
I don't understand why is this code in the interface definition

Reply:
because it is common to every backend? At least as long as we keep `right_click`, `left_click` and `both_click`?